scipy/sparse/csgraph/_shortest_path.pyx

```diff
@@ -430,22 +430,39 @@ def dijkstra(csgraph, directed=True, indices=None,
         will be equal to np.inf (i.e., not connected).
 
         .. versionadded:: 0.14.0
+    min_only: bool, optional
+￼        If False (default), for every node in the graph, find the shortest path to every node in indices.
+        If True, for every node in the graph, find the shortest path to any of the nodes in indices (which
+        can be substantially faster).
```

Comment:
Can you PEP8-ify these docstrings (<80 char)?

Reply:
done